src/test/java/uk/gov/companieshouse/pscverificationapi/controller/impl/PscVerificationControllerImplMergeIT.java

```diff
@@ -163,69 +135,38 @@ void updatePscVerificationWhenReplacingFields() throws Exception {
             .andExpect(jsonPath("$.updated_at", is(SECOND_INSTANT.toString())))
             .andExpect(jsonPath("$.data.company_number", is("REPLACED")))
             .andExpect(jsonPath("$.data.psc_notification_id", is(PSC_ID)))
-            .andExpect(jsonPath("$.data.relevant_officer.name_elements.forename",
-                is(NAME_ELEMENTS.getForename())))
-            .andExpect(jsonPath("$.data.relevant_officer.name_elements.other_forenames",
-                is(NAME_ELEMENTS.getOtherForenames())))
-            .andExpect(jsonPath("$.data.relevant_officer.name_elements.title",
-                is(NAME_ELEMENTS.getTitle())))
-            .andExpect(jsonPath("$.data.relevant_officer.name_elements.surname", is("REPLACED")))
-            .andExpect(jsonPath("$.data.relevant_officer.date_of_birth", is("1990-11-11")))
-            .andExpect(jsonPath("$.data.relevant_officer.is_director", is(false)))
-            .andExpect(jsonPath("$.data.relevant_officer.is_employee", is(false)))
             .andExpect(jsonPath("$.data.verification_details.uvid", is(UVID)))
             .andExpect(jsonPath("$.data.verification_details.verification_statements",
-                Matchers.containsInAnyOrder(RO_IDENTIFIED.toString(), RO_DECLARATION.toString(),
-                    RO_VERIFIED.toString())))
+                Matchers.containsInAnyOrder((INDIVIDUAL_VERIFIED.toString()))))
```

Comment:
Can be simplified to `Matchers.contains(INDIVIDUAL_VERIFIED.toString())`, meaning "is a collection of this one item and no more"